plugins/spi/SPIOutputTest.cpp

```diff
@@ -1173,13 +1173,13 @@ void SPIOutputTest::testIndividualWS2812bControl() {
   // get fake SPI data stream
   data = backend.GetData(0, &length);
   // this is the expected spi data stream:
-  const uint8_t EXPECTED1[] = { 0x92, 0x4D, 0x34,   //Pixel 1 Green (10)
-                                0x92, 0x49, 0x26,   //Pixel 1 Red (1)
-                                0x9B, 0x49, 0xA4,   //Pixel 1 Blue (100)
-                                0x92, 0x49, 0x24,   //Pixel 2 Green (0)
-                                0x92, 0x49, 0x24,   //Pixel 2 Red (0)
-                                0x92, 0x49, 0x24    //Pixel 2 Blue (0)
-                            };                       
+  const uint8_t EXPECTED1[] = { 0x92, 0x4D, 0x34,   // Pixel 1 Green (10)
+                                0x92, 0x49, 0x26,   // Pixel 1 Red (1)
+                                0x9B, 0x49, 0xA4,   // Pixel 1 Blue (100)
+                                0x92, 0x49, 0x24,   // Pixel 2 Green (0)
+                                0x92, 0x49, 0x24,   // Pixel 2 Red (0)
+                                0x92, 0x49, 0x24    // Pixel 2 Blue (0)
+                            };
```

Comment:
Can we align with the } above if you're going to leave it on it's own line please, like the one below.